department-of-veterans-affairs/va.gov-cms
# VACMS-11265: Reveal Linky destination to editor.

## Description

Closes #11265.

## Testing done
Local testing.

## Screenshots

https://user-images.githubusercontent.com/1318579/197853029-845bf6a4-47ea-482b-8c62-c0114ba41198.mov

## QA Steps
As any user capable of editing:

1. Browse to [the Managed Links page](https://cms-0x2qygiqnblqzee86kwgivpdlnlzn2fy.ci.cms.va.gov/admin/content/linky).
    - [x] Verify that there's a link to ArsTechnica (our test external link).
2. Open up [this node](https://cms-0x2qygiqnblqzee86kwgivpdlnlzn2fy.ci.cms.va.gov/test-data-my-test-office)
3. Click the Edit tab and expand the Content thing.
![Screen Shot 2022-10-25 at 3 14 23 PM](https://user-images.githubusercontent.com/1318579/197861850-708fa27e-3dff-4048-baf6-d0bd89f2773c.png)
    - [x] Click inside the External Link, click the "Link" button, and verify that the actual URL (not the Linky link) is displayed.
    - [x] Click inside the Internal Link, click the "Link" button, and verify that it's still linked to an entity and isn't doing weird stuff.
4. Update the External Link to point to another external site, preferably not a VA.gov website (for verification purposes).
5. Save the node.
6. Browse to [the Managed Links page](https://cms-0x2qygiqnblqzee86kwgivpdlnlzn2fy.ci.cms.va.gov/admin/content/linky).
    - [x] Verify that there's now a Linky pointing to the external site you specified.
    - [x] Verify the original linky destination exists. (it didn't overwrite the original)
